unimplemented/morse.py

```diff
@@ -0,0 +1,159 @@
+from typing import Optional, Literal, List
```

Comment:
probably should be removing it from ./unimplemented if morse is done now

Reply:
Removed rom ./unimplemented